authres_test.go

```diff
@@ -100,7 +127,8 @@ func TestParseAuthenticationResults(t *testing.T) {
 				t.Errorf("test %d: error: %v", n, err)
 			}
 			if res != nil && !reflect.DeepEqual(*res, tc.authres) {
-				t.Errorf("want: %+v, got: %+v", tc.authres, *res)
+				t.Errorf("want: %+v\n", tc.authres)
```

Comment:
I'd better leave `want: %v,\ngot: %v\n` here. It's an idiomatic way of doing assertion in tests.